cli/teos_cli.py

```diff
@@ -426,6 +461,12 @@ def main(args, command_line_conf):
                             if appointment_data:
                                 print(appointment_data)
 
+                    elif command == "get_all_appointments":
+                        get_all_appointments_endpoint = "{}/get_all_appointments".format(teos_url)
```

Comment:
Pass `teos_url` and move the endpoint construction inside the method for consistency with the rest.